### ISSUE TITLE
Change page menu to hamburger on smaller devices 

### ISSUE THREAD

**Issue by dekmeister** [dekmeister/discard-sealion#21]
The list of pages overflows on phones in the header menu. Below a minimum width it should be a hamburger dropdown menu for accessibility.
